Classification/include/CGAL/Classification/Feature/Elevation.h

```diff
@@ -99,43 +99,43 @@ class Elevation : public Feature_base
 
     for (std::size_t j = 0; j < grid.height(); ++ j)
       for (std::size_t i = 0; i < grid.width(); ++ i)
-      {
-        float mean = 0.;
-        std::size_t nb = 0;
-        typename Grid::iterator end = grid.indices_end(i,j);
-        for (typename Grid::iterator it = grid.indices_begin(i,j); it != end; ++ it)
+        if (grid.has_points(i,j))
         {
-          mean += float(get(point_map, *(input.begin()+(*it))).z());
-          ++ nb;
+          float mean = 0.;
```

Comment:
Any reason to use float here?

Reply:
Same reason why it's used everywhere else in the package: to save memory (classification is often launched on massive data sets and many structures are stored, using `float` can make a difference while a large precision is not needed when computing features).

Reply:
Except if I'm wrong it is just one variable and you loose a lost of precision for computing the mean. Maybe it is not important but I wanted to mention it.